static/scripts/rewards/web3/erc20-permit.ts

```diff
@@ -61,6 +67,8 @@ export function claimErc20PermitHandler(permit: Erc20Permit, provider: JsonRpcPr
       toaster.create("success", `Claim Complete.`);
       console.log(receipt.transactionHash); // @TODO: post to database
```

Comment:
nit: pls remove this `@TODO`

Reply:
@rndquu for nits you can apply a "suggestion" via the UI here and commit it directly which can save time for the contributor.